src/audio_call.c

```diff
@@ -834,6 +835,60 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
     print_err (self, error_str);
 }
 
+void cmd_bitrate(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[MAX_STR_SIZE])
+{
+    char *error_str;
+
+    if ( argc != 1 ) {
+        error_str = "Must have value!";
+        goto on_error;
+    }
+
+    if ( self->is_call == false ) {
+        error_str = "Must be in a call";
+        goto on_error;
+    }
+
+    const uint32_t bitrate = atoi(argv[1]);
+
+    if ( bitrate < 6 || bitrate > 510 ) {
+        error_str = "Bitrate falls outside of Opus spec";
+
+        while (true) {}
+
+        goto on_error;
+    }
+
+    TOXAV_ERR_BIT_RATE_SET error;
+    audio_bit_rate_status_cb(CallControl.av, CallControl.calls[self->num].out_idx, bitrate, -1, &error);
```

Comment:
fixed that in a second commit when I tested it, it worked with EchoBot when I called it, so I thought it was correct at the time